csrc/ir/internal_nodes.h

```diff
@@ -3399,4 +3399,43 @@ class PreprocessGroupedMatmulInputSf : public Expr {
   }
 };
 
+class BlockQuantizationOp : public Expr {
+ public:
+  using Expr::Expr;
+
+  BlockQuantizationOp(
+      IrBuilderPasskey,
+      Val* output_scales,
+      Val* output,
+      Val* input);
+
+  NVFUSER_DECLARE_CLONE_AND_CREATE
+
+  Val* blockScales() const {
+    return output(1);
```

Comment:
nit: This feels a bit odd since the constructor takes the scaling factor output first.

Reply:
That's totally fine. Please leave details as much as you know.